src/textual_dev/tools/diagnose.py

```diff
@@ -158,7 +158,9 @@ def _console() -> None:
 
 def diagnose() -> None:
     """Print information about Textual and its environment to help diagnose problems."""
-    print("<!-- This is valid Markdown, please paste the following directly in to a GitHub issue -->")
+    print(
+        "<!-- This is valid Markdown, please paste the following directly in to a GitHub issue -->"
```

Comment:
Given that this was edited, maybe we can also fix this typo? :P

```suggestion
        "<!-- This is valid Markdown, please paste the following directly into a GitHub issue -->"
```

Reply:
It wasn't really edited, just black doing its thing I believe. 

Perhaps PR the change given this PR is already merged and released?